src/components/Body.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './Body.css';
+import { Link } from 'react-router-dom';
+
+const Body = () => {
+  return (
+    <div className='body'>
+
+      <Link style={{textDecoration:'none'}} to='/beers'>
+        <div className='bodyCard'>
+          <img src={require('../images/Beers.png')} alt='bar'></img>
```

Comment:
The way you are loading images seems overcomplicated. 

If you place your `images` folder inside the `public` folder, you will be able to render the image by pointing straight to it
```
<img src='images/Beers.png' alt='bar'></img>

Reply:
I wasn't able to load images from public folder, a message appeared saying that I cannot use anything outside the src folder and after searching online on a couple of places they said that it's just easier to move the images inside the src folder, I would love to know how to load images from public